searchlib/src/vespa/searchlib/memoryindex/document_inverter.h

```diff
@@ -76,7 +75,7 @@ class DocumentInverter {
      * For each text and uri field in the document a task for inverting and adding that
      * field (using a field inverter) is added to the 'invert threads' executor, then this function returns.
      **/
-    void invertDocument(uint32_t docId, const document::Document &doc);
+    void invertDocument(uint32_t docId, const document::Document &doc, OnWriteDoneType on_write_done);
```

Comment:
Unrelated to this change, but we should take a pass at the documentation of this class to ensure it is up to date.